04-Configuration.md

```diff
@@ -0,0 +1,174 @@
+---
+title:     Configuring TGMPA
+menutitle: Configuration
+anchor:    configuration
+permalink: /configuration/
+---
+
+### Configuring TGMPA for use with your theme/plugin
+
+
+The **{{site.tgmpa.name}}** library has an easy to use API to reference plugins and set config options. Consider the code below (taken from `example.php`).
+
+**_N.B.: this code may be slightly out of date - the download package will always contain the most up-to-date version._**
```

Comment:
It should also be mentioned that the code is not complete as the strings are missing. To keep the code snippet shorter I would only have one example in the plugin array.